config.yml

```diff
@@ -431,10 +431,25 @@ nodes:
     fields:
       - name: new_name
         type: node
+        comment: |
+          Represents a GlobalVariableReadNode.
```

Comment:
We need some more information here:

* Are there any other possible nodes that go here?
* Are there any restrictions on what the new name can be?